# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 30d5dfedf252
# Full Hash: 30d5dfedf252f4f10a0d8948e86f4e57f1c4ccf9
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-11 09:21:34
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r fc3b4c5ee4b2 -r 30d5dfedf252 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Tue Sep 10 16:31:40 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Tue Sep 10 16:31:41 2024 +0000
@@ -6,6 +6,7 @@
 
 using mozilla::OriginAttributes from "mozilla/ipc/BackgroundUtils.h";
 using struct mozilla::void_t from "ipc/IPCMessageUtils.h";
+using struct nsID from "nsID.h";
 
 namespace mozilla {
 namespace dom {
@@ -44,13 +45,15 @@
                    int64_t expires,
                    nsString path,
                    int32_t sameSite,
-                   bool partitioned) returns (bool v);
+                   bool partitioned,
+                   nsID operationId) returns (bool waitForNotification);
 
   async DeleteRequest(nsString domain,
                       OriginAttributes attrs,
                       nsString name,
                       nsString path,
-                      bool partitioned) returns (bool v);
+                      bool partitioned,
+                      nsID operationID) returns (bool waitForNotification);
 
   async Close();
 